xmppserver/src/main/java/org/jivesoftware/openfire/net/StanzaHandler.java

```diff
@@ -98,6 +97,11 @@ public abstract class StanzaHandler {
      */
     protected PacketRouter router;
 
+    /**
+     * Domain of the local server and remote server or client.
+     */
+    private DomainPair domainPair;
```

Comment:
Isn't a DomainPair instance per definition server-to-server specific? I'm not sure if it should be used by clients, or, by extension, be used in this abstract class.

Reply:
Moved to ServerStanzaHandler